com.oracle.truffle.r.test.native/packages/altreprffitests/README.md

```diff
@@ -0,0 +1,18 @@
+# `altreprffitests` package
+This is a package for testing various features of the *altrep* framework.
+It contains as little native code as possible and tests some high-level features
+of altrep framework.
+
+## How it works
+This package provides `vec_wrapper` altrep class that represents very simple
+wrapper for whatever data it gets into its "constructor" `vec_wrapper.create_instance`.
+Moreover, one can specify which altrep methods should be overriden with
+`gen.*` parameters to the constructor.
+
+
+## See also
+Note that there also exist a different package for testing altrep instances
+called `altrep-class-tests`.
```

Comment:
The `altrep-class-tests` is not there yet, right? I thought that the difference would be that one of the packages would test "conformity" of a user provided ALTREP instance

Reply:
It is not in the `fastr` repository - it is in my private repository https://github.com/Akirathan/altrep_tests_package